File: Algorithms/Easy/58 Length of Last Word/Solution.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        arrS = []
        currString = ""

        for i in range(len(s)):
            if s[i] != " ":
                currString += s[i]
            else:
                if len(currString) > 0:
                    arrS.append(currString)
                currString = ""
        
        if len(currString) > 0:
            arrS.append(currString)

        return len(arrS[-1])

File: Algorithms/Easy/58 Length of Last Word/Solution.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int lengthOfLastWord(String s) { // Easy way -->
        String[] arrS = s.split(" ");

        return arrS[arrS.length - 1].length();
    }

    public int lengthOfLastWordAgain(String s) { // Sad way -->
        List<String> arrS = new ArrayList<>();
        StringBuilder str = new StringBuilder();

        for (int i = 0; i < s.length(); i++) {
            if (Character.isAlphabetic(s.charAt(i))) {
                str.append(s.charAt(i));
            } else {
                if (str.length() > 0) {
                    arrS.add(str.toString());
                    str.delete(0, str.length());
                }
            }
        }

        if (str.length() > 0) {
            arrS.add(str.toString());
        }

        return arrS.get(arrS.size() - 1).length();
    }
}